cadcetrans/cadcetrans/tests/data/namecheck.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version = "1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
We're starting from scratch with this application.  How about we dispense with cumbersome XML in favour of a more industry standard JSON or YAML.

Reply:
We are not starting from scratch. JCMT already uses it, GEMINI also did.